src/theme/components/index.ts

```diff
@@ -4,19 +4,21 @@ import { Checkbox } from "./Checkbox";
 import { Code } from "./Code";
 import { Divider } from "./Divider";
 import { Radio } from "./Radio";
+import { Tabs } from "./Tabs";
 import { makeTag } from "./Tag";
 
 interface ComponentsConfig {
   Tag: PackageTagConfig[];
 }
 
-export const makeComponents = (config: ComponentsConfig) => {
+export const makeComponents: any = (config: ComponentsConfig) => {
```

Comment:
why does this need an `any` now?

Reply:
Ahh we can probably remove this - I was getting a pretty annoying typescript error in my editor. I gave a basic annotation to suppress the error, but it's definitely not a real solution

<img width="495" alt="Screen Shot 2021-11-10 at 8 50 38 AM" src="https://user-images.githubusercontent.com/8749859/141156729-e2fe7f19-8395-448f-9251-d1f5436ecea9.png">